documents/order.md

```diff
@@ -275,6 +275,75 @@ Razorpay::Order.edit(orderId,para_attr)
 -------------------------------------------------------------------------------------------------------
 
 
+### View RTO/Risk Reasons
+
+```go
+orderId = "order_XXXXXXXXXXXXX1"
+
+Razorpay::Order.fetch(orderId).view_rto
+```
+**Parameters**
+
+| Name     | Type   | Description                         |
+|----------|--------|-------------------------------------|
+| orderId* | string | The unique identifier of an order to access the fulfillment information. |
+
+**Response:**
+```json
+{
+  "risk_tier": "high",
+  "rto_reasons": [
+    {
+      "reason": "short_shipping_address",
+      "description": "Short shipping address",
+      "bucket": "address"
+    },
+    {
+      "reason": "address_pincode_state_mismatch",
+      "description": "Incorrect pincode state entered",
+      "bucket": "address"
+    }
+  ]
+}
+```
+-------------------------------------------------------------------------------------------------------
+
+### Update the Fulfillment Details
+
+```rb
+para_attr = {
+  "payment_method": "upi",
+  "shipping": {
+    "waybill": "123456789",
+    "status": "rto",
+    "provider": "Bluedart"
+  }
+}
+
+Razorpay::Order.fetch("order_XXXXXXXXXXXXX1").edit_fulfillment(para_attr)
+```
+**Parameters**
+
+| Name     | Type   | Description                         |
+|----------|--------|-------------------------------------|
```

Comment:
please format the table

Reply:
formatted the table 

https://github.com/razorpay/razorpay-ruby/blob/1fcd3fa00ae1048e83a282d329e3f3d0bdf6b1cb/documents/order.md?plain=1#L327